hack/machineset.sh

```diff
@@ -159,15 +158,21 @@ get_azure_ms() {
     sku="2019-datacenter-with-containers-smalldisk"
     release="17763.6293.240905"   
   fi
+  local az=$(echo "$linuxWorkerSpec" | jq -r .providerSpec.value.zone | sed 's/null/""/g')
+  local region=$(echo "$linuxWorkerSpec" | jq -r .providerSpec.value.location)
+  local loadBalancer=$(echo "$linuxWorkerSpec" | jq -r .providerSpec.value.publicLoadBalancer)
```

Comment:
There is a discrepancy from the feature PR where the load balancer is getting fetch from a master node. It's best to keep them aligned.

https://github.com/openshift/windows-machine-config-operator/pull/2783/files#diff-d8f0ab1f9c046a20a7bd826451aa45a1dd6a9b2ddb2d8af0cdbbbfb7ea50ac62R113

Reply:
It looks like it is https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/openshift_windows-machine-config-operator/2783/pull-ci-openshift-windows-machine-config-operator-master-azure-e2e-operator/1906721606554619904/artifacts/azure-e2e-operator/gather-extra/artifacts/machines.json

Reply:
Makes sense. Thanks for the context.